third_party/2and3/requests/packages/urllib3/__init__.pyi

```diff
@@ -1,8 +1,8 @@
 from typing import Any
 from . import connectionpool
-from . import filepost
-from . import poolmanager
-from . import response
+from . import filepost as filepost
```

Comment:
OK, I updated this PR and the mypy PR accordingly (the 'as name' rule does not apply to child modules now).